src/global/ValueId.h

```diff
@@ -29,6 +29,7 @@ enum struct Datatype {
   VocabIndex,
   LocalVocabIndex,
   TextRecordIndex,
+  WordVocabIndex,
   Date,
   MaxValue = Date
```

Comment:
```suggestion
   Date,
  WordVocabIndex
  MaxValue = WordVocabIndex
```

That way the `checkIndexVersion` test probably works (we don't have to rebuild all indexes then).
Just try it out whether it works (of course you will have to rebuild your local indices, but those are small).